R/bind.R

```diff
@@ -176,6 +176,7 @@ NULL
 vec_rbind <- function(...,
                       .ptype = NULL,
                       .names_to = rlang::zap(),
+                      # TODO: confirm that this should be unchanged
```

Comment:
I left a comment like this in all the places that _could_ gain `"unique_quiet"` and `"universal_quiet"`, but have not *yet*. These functions don't expose `quiet` currently and `quiet = false` is basically hard-wired into them in various ways.